js/models/conversations.js

```diff
@@ -257,7 +267,7 @@
                 if (contact.isMe()) {
                     return [false, contact];
                 } else {
-                    return Promise.all([this.isUntrusted(), contact]);
+                    return Promise.all([contact.isUntrusted(), contact]);
```

Comment:
Indeed it is a separate bug. It isn't that we were returning the wrong results, it's that we were recomputing the set of trust values for all group members for every single member of the group.

It doesn't need to be `safeIsTrusted`, since that's called from inside `isTrusted`.